tests/test_audio.py

```diff
@@ -141,6 +141,16 @@ def test_flac_stereo_24_bit(self):
         else:
             self.assertSimilar(audio.get_raw_data()[:32], b"\x00\x00\x00\x00\x00\x00\xfe\xff\x00\x00\x02\x00\x00\x00\xfe\xff\x00\x00\x00\x00\x00\xff\x01\x00\x00\x02\xfc\xff\x00\xfe\x01\x00")
 
+    def test_incompatible_audio_file_error(self):
+        self.AUDIO_FILE_EN_MP3 = path.join(path.dirname(path.realpath(__file__)), "english.mp3")
```

Comment:
`self.AUDIO_FILE_EN_MP3` -> `AUDIO_FILE_EN_MP3`
`self` seems to be unnecessary.